client/state/data-layer/wpcom/me/sites/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Internal dependencies
+ */
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { receiveSites } from 'state/sites/actions';
+import {
+	SITES_REQUEST,
+	SITES_REQUEST_FAILURE,
+	SITES_REQUEST_SUCCESS,
+} from 'state/action-types';
+
+/**
+ * Object holding query parameters for request all sites (exported for testing)
+ */
+export const SITES_REQUEST_QUERY_PARAMS = {
+	site_visibility: 'all',
+	include_domain_only: true,
+	site_activity: 'active',
+	fields: 'ID,URL,name,capabilities,jetpack,visible,is_private,is_vip,icon,plan,jetpack_modules,single_user_site,is_multisite,options', //eslint-disable-line max-len
```

Comment:
could we maybe line these up one-per-line and alphabetize them? would that be a nicer way of showing what is being requested?

```js
…
site_activity: 'active',
fields: [
	'ID',
	'URL',
	'capabilities',
	'icon',
	'is_icon',
	'is_private',
	'is_vip',
	'jetpack',
	'jetpack_modules',
	…
].join(','),
options: [
	…
].join(','),
…
```